library/multipicker/src/main/java/im/vector/lib/multipicker/utils/MediaFileUtils.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.lib.multipicker.utils
+
+import android.content.Context
+import java.io.File
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+object MediaFileUtils {
```

Comment:
I am not sure, but I am always wondering about the RAM long life footprint of `object`. It's maybe negligible, hence, maybe it could be a class, taking the context as constructor `val` parameter, WDYT?

Reply:
or just convert to a simple `fun`.